packages/server/src/config.ts

```diff
@@ -26,7 +26,7 @@ const config = {
     permissions: [PERMISSIONS.CAN_MANAGE_USERS, PERMISSIONS.CAN_MANAGE_SCHEMAS, PERMISSIONS.CAN_VIEW_DOCUMENTS, PERMISSIONS.CAN_MANAGE_DOCUMENTS],
   },
   inviteOnly: Boolean(env.INVITE_ONLY || true),
-  ethNetwork: env.ETH_NETWORK || 'mainnet',
-  ethProvider: env.ETH_PROVIDER || 'https://mainnet.infura.io/v3/55b957b5c6be42c49e6d48cbb102bdd5',
+  ethNetwork: env.ETH_NETWORK || 'kovan',
```

Comment:
Since we are changing the default, do we need to modify the deployment configs to explictely use mainnet?